docs/crocks/List.md

```diff
@@ -0,0 +1,111 @@
+# List
+
+`List a`
+
+--
+
+--
+
+```js
+--
+```
+
+`List` exposes the following functions on the constructor and instance:
+
+| Constructor | Instance |
+|:---|:---|
+| [`empty`](#empty), [`fromArray`](#fromArray), [`of`](#of) | [`ap`](#ap), [`chain`](#chain), [`concat`](#concat), [`cons`](#cons), [`empty`](#empty), [`equals`](#equals), [`filter`](#filter), [`head`](#head), [`map`](#map), [`of`](#of), [`reduce`](#reduce), [`reject`](#reject), [`sequence`](#sequence), [`tail`](#tail), [`toArray`](#toArray), [`traverse`](#traverse), [`value`](#value) |
+
+## Constructors
+
+### empty
+
+`List m => () -> m ()`
+
+### fromArray
+
+`List m => a -> m a`
+
+### of
+
+`List m => a -> m a`
+
+## Instances
+
+## Instance
+
+### ap
+
+`List m => m (a -> b) ~> m a -> m b`
+
+### chain
+
+`List m => m a ~> (a -> m b) -> m b`
+
+### concat
+
+`List m => m a ~> m a -> m a`
+
+### cons
+
+`List m => m a ~> (b -> m b) -> m a`
```

Comment:
`cons` adds an element to the front of the `List` so this should be:

`List m => m a ~> a -> m a`